.ci/config/stage-types.yaml

```diff
@@ -168,55 +168,67 @@ engine-rest-jakarta-unit-compatibility-wildfly:
   jdkVersion: 'openjdk-jdk-17-latest'
 platform-jdk-openjdk-jdk-17-latest:
   directory: '.'
-  command: 'install source:jar source:test-jar -pl ''!webapps'',''!webapps/assembly'',''!webapps/assembly-jakarta'' -Pdistro,distro-ce,distro-wildfly'
+  command: 'install source:jar source:test-jar -pl ''!webapps/assembly'',''!webapps/assembly-jakarta'' -Pdistro,distro-ce,distro-wildfly'
   stash:
     runtimeStash: true
   nodeLabel: 'h2'
   jdkVersion: 'openjdk-jdk-17-latest'
   withNpm: true
 platform-jdk-openjdk-jdk-11-latest:
   directory: '.'
-  command: 'install source:jar source:test-jar -pl ''!webapps'',''!webapps/assembly'',''!webapps/assembly-jakarta'' -Pdistro,distro-ce,distro-wildfly'
+  command: 'install source:jar source:test-jar -pl ''!webapps/assembly'',''!webapps/assembly-jakarta'' -Pdistro,distro-ce,distro-wildfly'
   stash:
     runtimeStash: true
   nodeLabel: 'h2'
   jdkVersion: 'openjdk-jdk-11-latest'
   withNpm: true
 platform-jdk-openjdk-jdk-8-latest:
   directory: '.'
-  command: 'install source:jar source:test-jar -pl ''!webapps'',''!webapps/assembly'',''!webapps/assembly-jakarta'',''!quarkus-extension/engine/deployment'',''!quarkus-extension/engine/runtime'',''!quarkus-extension/engine/qa'' -Pdistro,distro-ce,distro-wildfly'
+  command: 'install source:jar source:test-jar
+    -pl ''!distro/wildfly,!distro/wildfly/subsystem,!distro/wildfly/modules,!distro/wildfly/assembly,!distro/wildfly/webapp,!distro/wildfly/distro,
+          !qa/integration-tests-engine-jakarta,!webapps/assembly,
+          !quarkus-extension/engine/deployment,!quarkus-extension/engine/runtime,!quarkus-extension/engine/qa'' 
+    -Pdistro,distro-ce,distro-wildfly'
```

Comment:
👍 I like the formatting and reduction of quotes. The exclusion is long but in case the version gets dropped, the stages will be removed.